Scripts/SoundPlayer.cs

```diff
@@ -0,0 +1,88 @@
+using Godot;
+using System;
+
+public partial class SoundPlayer : AudioStreamPlayer
+{
+	private static readonly AudioStream buttonClick = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Button.mp3");
+	private static readonly AudioStream alien = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/CharacterSelect.mp3");
+	private static readonly AudioStream buy = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Buy.mp3");
+	private static readonly AudioStream eating = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Eating.mp3");
+	private static readonly AudioStream gameOver = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/game_over.mp3");
+	private static readonly AudioStream jump = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Jump.mp3");
+	private static readonly AudioStream memoryWin = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/MemoryFinish.mp3");
+	private static readonly AudioStream move = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/PlantMove.mp3");
+	private static readonly AudioStream trash = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Trash.mp3");
+	private static readonly AudioStream dry = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Die.mp3");
+	private static readonly AudioStream water = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Water.mp3");
+	private static readonly AudioStream overwater = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Water2.mp3");
+	private static readonly AudioStream flip = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Flip.mp3");	
+	private static readonly AudioStream pack = (AudioStream) GD.Load("res://Sounds/Effects/Buttons/Pack.mp3");
+	private static readonly AudioStream boom = (AudioStream) GD.Load("res://Sounds/Effects/Easteregg.mp3");
+
+	private void _play_sound(AudioStream music, float volume = 0.0f)
+	{
+		Stream = music;
+		VolumeDb = volume;
+		Play();
+	}
+	public void PlayButtonCick()
```

Comment:
Minor typo! You would need to also fix all references to this function in the code.
```suggestion
	public void PlayButtonClick()
```

Reply:
No i dont want to!!! (tehe)

Reply:
/kick